profile/README.md

```diff
@@ -14,9 +14,19 @@ There is currently an implementation effort working on creating the first versio
 <!--
 - A "Getting involved" section that explains which public, archived, linkable communication channels the project uses. This should include a link to the project issue tracker, but also to any further discussion media used.
 -->
+If you want to get involved in the project with us, you can take a look at the [Registry project's Readme](https://github.com/project-origin/registry#readme), or at the [Partner Collaboration Readme](https://github.com/project-origin/origin-collaboration#readme) for an overview of what we are doing right now. 
+
+For a deeper look into the progress and ongoing tasks in this project, you can take a look at our [Organization Project Boards](https://github.com/orgs/project-origin/projects), [Registry Project Boards](https://github.com/project-origin/registry/projects?query=is%3Aopen), or finally, if you are part of our Internal collaboration, you should chek out the [Collaboration Project Boards](https://github.com/project-origin/origin-collaboration/projects?query=is%3Aopen) 
```

Comment:
Typo.

```suggestion
For a deeper look into the progress and ongoing tasks in this project, you can take a look at our [Organization Project Boards](https://github.com/orgs/project-origin/projects), [Registry Project Boards](https://github.com/project-origin/registry/projects?query=is%3Aopen), or finally, if you are part of our Internal collaboration, you should check out the [Collaboration Project Boards](https://github.com/project-origin/origin-collaboration/projects?query=is%3Aopen) 
```

Same point of the three words on the "internal collaboration" thing at a later point in time. Goes along with the usual "boundaries", "growing from user, contributor, to trusted committer" and governance discussions. Let's not forget that, but not deal with it right now.